testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -332,15 +396,15 @@ def test_results_type(u):
     (20, 50, 2, 15),
     (20, 50, None, 30)
 ])
-def test_AnalysisFromFunction(u, start, stop, step, nframes):
+def test_AnalysisFromFunction(u, start, stop, step, nframes, client_AnalysisFromFunction):
```

Comment:
Add a comment about location of definition for `client_AnalysisFromFunction` and purpose.

Once there are a few of these comments, someone new to the tests will pick up the pattern.